public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -273,8 +273,8 @@ define(function(require) {
 
     // Expose the published state with the updated link
     published.link
-      .attr("href", publishUrl)
-      .text(publishUrl);
+      .attr("href", "https://" + publishUrl)
+      .text("https://" + publishUrl);
```

Comment:
@humphd I will see what I can do with window.location.protocol. 
Thank you for your suggestion! 👍